opentelemetry-sdk/tests/metrics/integration_test/test_console_exporter.py

```diff
@@ -20,18 +22,76 @@
     ConsoleMetricExporter,
     PeriodicExportingMetricReader,
 )
+from opentelemetry.test.globals_test import reset_metrics_globals
 
 
 class TestConsoleExporter(TestCase):
     def test_console_exporter(self):
 
-        try:
-            exporter = ConsoleMetricExporter()
-            reader = PeriodicExportingMetricReader(exporter)
-            provider = MeterProvider(metric_readers=[reader])
-            metrics.set_meter_provider(provider)
-            meter = metrics.get_meter(__name__)
-            counter = meter.create_counter("test")
-            counter.add(1)
-        except Exception as error:
-            self.fail(f"Unexpected exception {error} raised")
+        reset_metrics_globals()
```

Comment:
Maybe put this in a setUp and tearDown so it doesn't leak into any tests running after

Reply:
fixed